src/core_write.cpp

```diff
@@ -224,9 +236,21 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
         ScriptPubKeyToUniv(txout.scriptPubKey, o, true);
         out.pushKV("scriptPubKey", o);
         vout.push_back(out);
+
+        // Sanity check to avoid overflow
+        if (MoneyRange(txout.nValue)) {
+            nTotalOut += txout.nValue;
+        }
```

Comment:
This won't work right... `nTotalOut` would just be missing part of its amount, and the result calculated later will still be wrong.